connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -441,11 +441,14 @@ class SparkConnectPlanner(session: SparkSession) {
           case x => UnresolvedAlias(x)
         }
 
+    // Retain group columns in aggregate expressions:
+    val aggExprs =
+      groupingExprs ++ rel.getResultExpressionsList.asScala.map(transformResultExpression)
```

Comment:
To match DataFrame API, I think we should allow clients to choose the behavior. How about we add a bool flag in the proto message of `Aggregate`? Clients can either use config or expose APIs directly to set this bool flag.

Reply:
@cloud-fan from your experience how often have you seen someone to explicitly set the flag since the default was changed in Spark 2.x? I don't think it's worth it adding the full concept of session options and using this as the prime example. 

Our goal should be to simplify, and the users can still override the defaults on the server.

Reply:
I don't have data points, but people can always add a `.select(...)` to exclude grouping columns. I'm fine to always include grouping columns here.